activiti-cloud-services-metadata/README.md

```diff
@@ -0,0 +1,10 @@
+# App Metadata
+
+A way to get at metadata such as:
+
+activiti.cloud.service.version
+activiti.cloud.service.type
+activiti.cloud.application.name
+activiti.cloud.application.version
+
+Each app publishes to eureka.
```

Comment:
yes.. but not only eureka.. so I would change that to service registry. And this is not actually doing that.. meaning that we have the one that does the registration to the ServiceInstance, so this should be agnostic and it should only allow us to autowired this into the application that uses the dependency.